opentelekomcloud/acceptance/dws/resource_opentelekomcloud_dws_cluster_v1_test.go

```diff
@@ -0,0 +1,154 @@
+package acceptance
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/dws/v1/cluster"
+
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/common"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/acceptance/env"
+	"github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/common/cfg"
+)
+
+const resourceInstanceName = "opentelekomcloud_dws_cluster_v1.cluster_1"
+
+func TestAccDwsClusterV1_basic(t *testing.T) {
+	var cls cluster.ClusterDetail
+	var clusterName = fmt.Sprintf("dws_cluster_%s", acctest.RandString(5))
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { common.TestAccPreCheck(t) },
+		ProviderFactories: common.TestAccProviderFactories,
+		CheckDestroy:      testAccCheckDwsV1ClusterDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccDwsV1ClusterBasic(clusterName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDwsV1ClusterExists(resourceInstanceName, &cls),
+					resource.TestCheckResourceAttr(resourceInstanceName, "name", clusterName),
+					resource.TestCheckResourceAttr(resourceInstanceName, "number_of_node", "3"),
+				),
+			},
+			// {
+			// 	Config: testAccDwsV1ClusterUpdated(clusterName),
+			// 	Check: resource.ComposeTestCheckFunc(
+			// 		resource.TestCheckResourceAttr(resourceInstanceName, "name", clusterName),
+			// 		resource.TestCheckResourceAttr(resourceInstanceName, "number_of_node", "6"),
```

Comment:
not working?

Reply:
uncommented, it is working but not often